notebooks/write_MexHrscPds3LabelNaifSpiceDriver.ipynb

```diff
@@ -138,190 +64,40 @@
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "def ecef_to_lla(x, y, z, a_radius, b_radius):\n",
-    "    \"\"\" Converts from earth-centric, earth-fixed to lat, lon, altitude\n",
-    "    \"\"\"\n",
-    "    ecef = pyproj.Proj(proj='geocent', a=a_radius, b=b_radius)\n",
-    "    lla = pyproj.Proj(proj='latlong',  a=a_radius, b=b_radius)\n",
+    "# Save off the json and read it back in to check if\n",
+    "# the json exists and was formatted correctly\n",
+    "# json_file = os.path.splitext(fileName)[0] + '.json'\n",
+    "# with open(json_file, 'w') as fp:\n",
+    "#     json.dump(usgscsm_dict, fp)\n",
     "    \n",
-    "    lon, lat, alt = pyproj.transform(ecef, lla, x, y, z)\n",
-    "    return lon, lat, alt"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "image_pt = csmapi.ImageCoord(.5, .5)\n",
-    "ecef_coord = camera.imageToGround(image_pt, 0.0)\n",
-    "ecef_coord.x, ecef_coord.y, ecef_coord.z\n",
-    "a_rad, b_rad = knoten.csm.get_radii(camera)\n",
-    "lon, lat, _ = ecef_to_lla(ecef_coord.x, ecef_coord.y, ecef_coord.z, a_rad, b_rad)\n",
-    "lon = (lon+360)%360"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "pvl_output = pvl.loads(isis.campt(from_=cub_loc, line=1, sample=1))"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Grab spacecraft position and body fixed look vector from csm\n",
-    "locus = camera.imageToRemoteImagingLocus(csmapi.ImageCoord(1 - .5, 1 - .5))\n",
-    "csm_bodyfixedLV = np.asarray([locus.direction.x, locus.direction.y, locus.direction.z])\n",
-    "csm_position = np.asarray([locus.point.x, locus.point.y, locus.point.z])\n",
-    "\n",
-    "# Grab spacecraft position and body fixed look vector from campt pvl output\n",
-    "campt_bodyfixedLV = np.asarray(pvl_output['GroundPoint']['LookDirectionBodyFixed'])\n",
-    "campt_position = pvl_output['GroundPoint']['SpacecraftPosition']\n",
-    "campt_position = np.asarray(campt_position.value) * 1000\n",
-    "\n",
-    "# Compute the differences\n",
-    "print(csm_bodyfixedLV - campt_bodyfixedLV)\n",
-    "print(csm_position - campt_position)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "def check_pixel(camera, cub, line, sample):\n",
-    "    \"\"\"Compares ISIS and USGSCSM pixel.\n",
+    "# with open(json_file, 'r') as fp:\n",
+    "#     usgscsm_dict = json.load(fp)\n",
```

Comment:
Is there a reason for all of these changes to the notebook?

Reply:
I updated the notebook with the new way we are selecting metakernels and outputting the json from the formatter functions. I can undo these changes if we do not want to update the notebooks yet.

Reply:
The changes are good then. Can you remove the old commented out code.